README.md

```diff
@@ -1,14 +1,38 @@
-# Project
+# Azure Migrate application and code assessment toolkit
```

Comment:
@brunoborges , can you please update the title to "Azure Migrate application and code assessment for Java"